src/key.h

```diff
@@ -101,6 +101,20 @@ class CKey
     //! Negate private key
     bool Negate();
 
+    //! Tweak a secret key by adding a scalar value to it.
+    //
+    CKey AddTweak(const unsigned char *tweak32) const;
+
+    //! Tweak a secret key by multiplying it by a scalar value.
+    CKey MultiplyTweak(const unsigned char *tweak32) const;
```

Comment:
Same as other suggestion
```suggestion
    CKey AddTweak(const uint256& tweak32) const;

    //! Tweak a secret key by multiplying it by a scalar value.
    CKey MultiplyTweak(const uint256& tweak32) const;
```